### PR TITLE
Feature/export

### PR DESCRIPTION
**Pull request by lucbpz**
## What does this PR do?

- Adds the new export dialog to export a volume into a file, a local image and a new image.

image

### CODE REVIEW

File: ui/src/components/ExportDialog.tsx [hunk 1]
```diff
@@ -1,132 +1,246 @@
...
+              fullWidth
+              defaultValue={`${context.store.volume.volumeName}.tar.gz`}
+              spellCheck={false}
+              onChange={(e) => {
+                setFileName(e.target.value);
+              }}
+            />
+            <Grid container alignItems="center" gap={2}>
+              <Grid item flex={1}>
+                <TextField
```
Comment (felipecruz91) [inline ui/src/components/ExportDialog.tsx:115]: `TextField` is empty _after_ having selected the directory from the file picker. I'd like to see what directory has been selected.

File: ui/src/components/ExportDialog.tsx [hunk 2]
```diff
@@ -1,132 +1,246 @@
...
+interface Props {
+  open: boolean;
+  onClose(v: boolean): void;
 }
 
-export default function ExportDialog({...props}) {
-    console.log("ExportDialog component rendered.");
-    const ddClient = useDockerDesktopClient();
-    const context = useContext(MyContext);
+export default function ExportDialog({ open, onClose }: Props) {
```
Comment (felipecruz91) [inline ui/src/components/ExportDialog.tsx:36]: Previously, when saving an image, we had a nice message explaining to users how they could check that the content had been saved successfully in the image.

I think it'd be good to keep it when saving a volume into both a local and a new image, WDYT?

image

File: ui/src/components/ExportDialog.tsx (hunk 1 above)
Comment (lucbpz) [reply ui/src/components/ExportDialog.tsx:116]: solved in [d0ea291](https://github.com/felipecruz91/vackup-docker-extension/pull/34/commits/d0ea291a948f729256acca5ef57c721c75d69c4c)

Comment (felipecruz91) [reply ui/src/components/ExportDialog.tsx:116]: Works like a charm now, thanks!

**Pull request merged**